metricbeat/modules.d/kubernetes.yml.disabled

```diff
@@ -28,6 +37,8 @@
 #    - state_container
 #  period: 10s
 #  hosts: ["kube-state-metrics:8080"]
+#  add_metadata: true
```

Comment:
Not introduced in this PR, but I think commenting with `#` should be smae as first part of config (line 16 etc).